public/app/features/alerting/unified/hooks/useAbilities.test.tsx

```diff
@@ -213,6 +217,97 @@ describe('AlertRule abilities', () => {
   });
 });
 
+describe('enrichment abilities', () => {
+  setupMswServer();
+
+  it('should grant read and write permissions to admin users', () => {
+    grantPermissionsHelper([]);
+    jest.spyOn(require('../utils/misc'), 'isAdmin').mockReturnValue(true);
```

Comment:
nit: you could try `import * as misc from '../utils/misc`
Then `jest.spyOn(misc, 'admin')`
I think it would give you better type safety than `require`